bootstrap/main.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import pathlib
+import shutil
+import sys
+import time
+from typing import Any
+
+import docker
+
+client = docker.from_env()
+
+default_file_path = pathlib.Path("/startup.json.default")
+config_file_path = pathlib.Path("/config/startup.json")
+
+CORE_CONTAINER_NAME = "companion_core"
+
+
+def ensure_dir(file_path: str) -> None:
+    """Makes sure that the file path exists
+
+    Args:
+        file_path (str): path to check/make
+    """
+    directory = os.path.dirname(file_path)
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+
+def stop_old_container() -> None:
+    """ Looks for an old core container and stops it if found"""
+    try:
+        old_container = client.containers.get(CORE_CONTAINER_NAME)
+        old_container.stop()
+        old_container.remove()
+    except Exception:
+        pass
+
+
+def overwrite_config_file_with_defaults() -> None:
+    """Overwrites the config file with the default configuration"""
+    shutil.copy(default_file_path, config_file_path)
+
+
+def read_config_file() -> Any:
```

Comment:
This should return a json, and from what I see in your json file is a `Dict[str, Any]` right ?

Reply:
it is more of a Dict[str, Dict[str, Any]]. But after some tries and reading [this](https://github.com/python/typing/issues/182)  I thought it was just not worth the extra clutter. I'll give it another try.

Reply:
Yep, `Dict[str, Any]` will not be the most appropriate and correct way, but I believe that is much better than `Any`.